src/test/java/com/bookshop/repository/BookRepositoryTests.java

```diff
@@ -20,26 +21,95 @@ public class BookRepositoryTests {
     @Autowired
     private BookRepository bookRepository;
 
-    //@Test
-    @DisplayName("Find only book with category 1")
-    @Sql(scripts = "classpath:database/books/add-one-book-with-first-category.sql",
+    @Test
+    @DisplayName("Find only book with correct category 1")
+    @Sql(scripts = {"classpath:database/books/remove-all-books-and-categories.sql",
+            "classpath:database/books/add-one-book-with-first-category.sql"},
             executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
     @Sql(scripts = "classpath:database/books/remove-all-books-and-categories.sql",
             executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
     public void findAllByCategoryId_OneBookWithCorrectCategory_ReturnsOneBook() {
```

Comment:
```suggestion
    public void findAllByCategoryId_oneBookWithCorrectCategory_returnsOneBook() {
```
As far as I know, this is a common practice (if you agree, then please change it in all methods)